draft-ietf-acme-acme.md

```diff
@@ -2111,11 +2164,22 @@ keyAuthorization (required, string):
 from the challenge and the client's account key.
 
 ~~~~~~~~~~
-/* BEGIN JWS-signed content */
+POST /acme/authz/1234/2
+Host: example.com
+Content-Type: application/jose+json
+
 {
-  "keyAuthorization": "evaGxfADs...62jcerQ"
+  "protected": base64url({
+    "alg": "ES256",
+    "kid": "https://example.com/acme/acct/1",
+    "nonce": "JHb54aT_KTXBWQOzGYkt9A",
+    "url": "https://example.com/acme/authz/1234/1"
```

Comment:
I think this `"url"` should be `https://example.com/acme/authz/1234/2` instead of `/1`. Yes?